backends/bmv2/run-bmv2-test.py

```diff
@@ -265,6 +285,22 @@ def main(argv):
             else:
                 options.compilerOptions += argv[1].split();
                 argv = argv[1:]
+        elif argv[0] == "--sa":
```

Comment:
If you use a separate script to drive your dynamic loading test case, then none of these command line arguments are needed.

Reply:
Please see my responses above.

Reply:
Your test case does not use `--sa` option, how is it different from `--stsa`? What about changing it to `-s` for passing option to bmv2 switch? the `stsa` acrynym is difficult to understand and remember.

Reply:
The syntax for invoking simple_switch is: ./simple_switch <general arguments> -- <target-specific arguments>. --sa is used to provide general arguments (Switch Arguments). --stsa is used for target-specific arguments (Switch Target Specific Argumenta). Even though --sa isn't used here, I included it for completeness.

Reply:
Sorry, I see github swallowed the syntax. I meant to say it's:
./simple_switch \<general arguments\> -- \<target-specific arguments\> .

Reply:
The `--sa` acronym is difficult to understand, can you spell it out please?